src/js/constants/ServiceSchema.js

```diff
@@ -2,64 +2,13 @@
 import React from 'react';
 /* eslint-enable no-unused-vars */
 
+import General from './service-schema/general';
```

Comment:
In general (no I don't mean the var name itself ;) ) the letter casing should follow the file casing.
So I suggest to rename the file `general` to `General`.